pulumi/typescript_integration_tests/__main__.py

```diff
@@ -99,7 +99,7 @@ def require_str(key: str) -> str:
         #     NomadServicePostgresDbArgs, ref.require_output("plugin-work-queue-db")
         # )
 
-        # self.test_user_password_secret_id = require_str("test-user-password-secret-id")
+        self.test_user_password_secret_id = require_str("test-user-password-secret-id")
```

Comment:
If you've incorporated changes from `main`, you don't need this entire `GraplStack` implementation; you can just pull in the common one, like we do in the `python_integration_tests` and `rust_integration_tests` Pulumi projects.